oioioi/default_settings.py

```diff
@@ -217,6 +217,7 @@
     'oioioi.workers',
     'oioioi.quizzes',
     'oioioi._locale',
+    'oioioi.mp',
```

Comment:
I am not sure all oioioi instances should have this module enabled by default. I would follow the convention with simillar `oioioi.pa` module which does not appear on this list.